README.md

```diff
@@ -41,20 +41,20 @@ let config = TokenConfiguration(token: "12345")
 or for GitHub Enterprise
 
 ```swift
-let config = TokenConfiguration("12345", url: "https://github.example.com/api/v3/")
+let config = TokenConfiguration("2beef4ace", url: "https://github.example.com/api/v3/")
```

Comment:
This seems not needed

Reply:
just more clear that it's token, not the digits